src/Bridge/Symfony/Bundle/Test/Constraint/ArraySubset.php

```diff
@@ -13,94 +13,15 @@
 
 namespace ApiPlatform\Core\Bridge\Symfony\Bundle\Test\Constraint;
 
-use PHPUnit\Framework\Constraint\Constraint;
 use PHPUnit\SebastianBergmann\Comparator\ComparisonFailure;
 use SebastianBergmann\Comparator\ComparisonFailure as LegacyComparisonFailure;
 
 if (!class_exists(ComparisonFailure::class)) {
-    class_alias(LegacyComparisonFailure::class, ComparisonFailure::class);
+    class_alias(LegacyComparisonFailure::class, 'PHPUnit\SebastianBergmann\Comparator\ComparisonFailure');
```

Comment:
You can keep using ::class even if the class doesn't exist.

Reply:
phpstan does not agree

Reply:
When using `::class`, phpstan loads the class and fails because it doesn't find it.